docs/api/ctx.md

```diff
@@ -17,10 +17,10 @@ Based on the request’s [Accept](https://developer.mozilla.org/en-US/docs/Web/H
 :::
 
 ```go title="Signature"
-func (c *Ctx) Accepts(offers ...string)          string
-func (c *Ctx) AcceptsCharsets(offers ...string)  string
-func (c *Ctx) AcceptsEncodings(offers ...string) string
-func (c *Ctx) AcceptsLanguages(offers ...string) string
+func (c Ctx) Accepts(offers ...string)          string
+func (c Ctx) AcceptsCharsets(offers ...string)  string
+func (c Ctx) AcceptsEncodings(offers ...string) string
+func (c Ctx) AcceptsLanguages(offers ...string) string
```

Comment:
Should be (c *DefaultCtx) i think

Reply:
I had assumed that the documentation would now refer to the interface provided in [`ctx_interface.go`](https://github.com/gofiber/fiber/blob/main/ctx_interface.go#L22) for v3 users, though I wasn't sure.

It looks like the `v2` docs used to refer to [`*Ctx`](https://github.com/gofiber/fiber/blob/v2/ctx.go#L199), which now was directly renamed to [`*DefaultCtx`](https://github.com/gofiber/fiber/blob/main/ctx.go#L199).

I can make this change to be `*DefaultCtx` though to match [`ctx.go`](https://github.com/gofiber/fiber/blob/main/ctx.go#L199) instead. Please let me know what you think. @efectn

Reply:
I've added a commit to make it clear that `/docs/api/ctx.md` is now referring to the Ctx interface, rather than a pointer to the Ctx struct (which is now DefaultCtx).

Since this documentation is for users, I figured that it would make most sense to use the interface that they will access directly rather than the underlying `DefaultCtx` implementation that Fiber uses internally.

I've re-requested for review. Please let me know what you think about this.

Reply:
@gofiber/maintainers what do you think
```go
func (c *DefaultCtx)
```
or 
```go
func (c Ctx)
```